docs/v0.6/en/user-guide/client-libraries/go.md

```diff
@@ -25,9 +25,10 @@ Import the library in your code:
 
 ```go
 import (
-    greptime "github.com/GreptimeTeam/greptimedb-client-go"
-    "google.golang.org/grpc"
-    "google.golang.org/grpc/credentials/insecure"
+    "github.com/GreptimeTeam/greptimedb-ingester-go/client"
```

Comment:
Please use indent. This is Golang's format. You can compare with lines below and copy from there.